omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -147,6 +147,9 @@ public Object upload(MultipartFile file, RequestContext context) throws Response
 				obs = obsSaver.saveImageAttachment(visit, patient, encounter, fileCaption, file, instructions);
 				break;
 			
+			case EXECUTABLE:
+				throw new IllegalRequestException("File type is not allowed as attachment.");
+				
```

Comment:
Rather than throwing an exception here, it might just be better to silently do nothing. It is worth considering if we can add the same type exclusions to the front-end code so that users who (for whatever reason) are trying to use the UI to upload an invalid attachment get some kind of useful error message.

Reply:
Hello @ibacher, That was my imagination at the first place.  But it seams that validating from here  was better.  
If i remove the exception, then then what should i do to notify the frontend that the type is vulnerable

Reply:
@ibacher isn't there a way to do what you say by dealing or not dealing with the backend error at the frontend?

Or this will literally blow things up with a code 500 of sorts?

Reply:
@mks-d We could do something with the 500 error on the frontend. But that won't do much for a malicious actor who directly hits the REST API. That said, it's not a big deal either way.